internal/spec/endpoint.yml

```diff
@@ -2,6 +2,8 @@ name: Endpoint Security
 cmd: endpoint-security
 artifact: endpoint-dev
 service: 6788
+process:
+    stop_timeout: 180
```

Comment:
Can we add a comment explaining why this value was chosen? Maybe just link to https://github.com/elastic/elastic-agent/issues/1262 and mention that the value was chosen based on input from the endpoint team?